projects/angular/components/ui-grid/src/ui-grid.component.scss

```
@@ -240,6 +240,25 @@ ui-grid {
                 display: flex;
                 margin-left: auto;
                 align-items: center;
+                overflow: hidden;
+
+                .ui-grid-action-buttons {
+                    overflow: hidden;
+
+                    button {
```

If there are any icon buttons in the rhs side, this would apply to them too. Also, anchor buttons might be used (we have a few cases), where the element's tag is `a`, it wouldn't apply to them. 

```suggestion
                    .mdc-button {
```

---

First, I reverted this commit, so it doesn't matter for now.
Second, regarding the anchor tag, I agree; I noticed that while testing and I updated the code here but because I rolled back these changes they it didn't reach git remote. Also, not sure I get what you mean with the button part. Does an icon button not contain a button element?

---

As above, to be resolved by: https://uipath.atlassian.net/browse/OR-75789